apps/public-docsite-v9/src/Concepts/Positioning/PositioningShiftToCoverTarget.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import {
+  Button,
+  makeStyles,
+  SpinButton,
+  Label,
+  Menu,
+  MenuTrigger,
+  MenuPopover,
+  MenuList,
+  MenuItem,
+} from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  boundary: {
+    border: '2px dashed red',
+    width: '300px',
+    height: '300px',
+    overflow: 'auto',
+  },
+  trigger: {
+    display: 'block',
+    width: '150px',
+    margin: '200px auto',
+  },
+});
+
+export const CoverTargetForSmallViewport = () => {
```

Comment:
Let's add a toggle to switch to an example with horizontal main axis and update the width of a container

Reply:
I added an option to toggle menu position, and made the boundary resizable:
<img width="424" alt="Screenshot 2024-12-16 at 16 01 39" src="https://github.com/user-attachments/assets/97a14173-f5ff-4e4a-bb70-d8e0fbd7636b" />